src/fsharp/SyntaxTree.fsi

```diff
@@ -810,7 +823,7 @@ type SynExpr =
     /// F# syntax: ident
     /// Optimized representation for SynExpr.LongIdent (false, [id], id.idRange)
     | Ident of
-        ident: Ident
+        ident: IdentWithTrivia
```

Comment:
I think this one should still be Ident, and SynExpr.Ident should only be used when there is no trivia.  It is meant to be an optimized common-case representation and the norm is there is no trivia.